### DISCUSSION TITLE
hls proxyfing

### DISCUSSION THREAD

**Discussion by malletfils** [bluenviron/mediamtx#355] [Questions]
Hello,

I'd like to integrate a m3u8 stream from a camera to proxify it and serve it as a RTSP stream from RTSP simple server. Is this possible ?  The URL I've got for the m3u8 is http, and sourceRedirect asks for a rtsp URL.

Thanks for your help

**Comment by malletfils** [comment]
Ok, forget about it. The fact is that my cam sends a AAC audio stream with no global headers, and this is not supported. Removing the audio stream from the FFMPEG call solved the problem (unfortunately, I have no sound but...). Thanks

**Reply by aler9** [reply]
well, anyway HLS support is coming - reading with HLS is supported in main, and proxying HLS is coming soon.

**Reply by aler9** [reply]
> The fact is that my cam sends a AAC audio stream with no global headers, and this is not supported

It is supported through the RTP MP4A-LATM profile, that can be enabled with `rtpflags`

```
ffmpeg -re -i http://.. -c copy -rtpflags latm -f rtsp rtsp://...
```

**Reply by aler9** [reply]
This feature is now available in v0.17.3

[a comment by an automated account is not shown]
